lib/galaxy/schema/schema.py

```diff
@@ -338,6 +342,15 @@ class DetailedUserModel(BaseUserModel, AnonUserModel):
     tags_used: List[str] = Field(default=Required, title="Tags used", description="Tags used by the user")
 
 
+class UserCreationPayload(Model):
+    password: str = Field(default=Required, title="user_password", description="The password of the user.")
+    remote_user_email: Optional[str] = Field(
```

Comment:
I think this might be the only required field when `use_remote_user` is set in the Galaxy config, see https://bioblend.readthedocs.io/en/latest/api_docs/galaxy/all.html#bioblend.galaxy.users.UserClient.create_local_user and https://bioblend.readthedocs.io/en/latest/api_docs/galaxy/all.html#bioblend.galaxy.users.UserClient.create_remote_user

I would separate this into two alternative models and then create a union for valid payloads.